### DISCUSSION TITLE
Case of use and technology

### DISCUSSION THREAD

**Discussion by wilonweb** [smartcontractkit/full-blockchain-solidity-course-py#575] [General]
Hello, I heard Patrick Collins say, JS is good for front and Python is the best for smartContract. 
I would like to have differents opinions about Python, Solidity, JS for a Eth use.
Are some better depending on the use case ? 
Or 
Is the best choice and the language we master ?

**Comment by cromewar** [comment]
Hello @NinjaTurtles-cloud well it depends, not on the technology but  on people.
People who like JS will say it's better and people who like Python will say it's better, So you should use whatever you feel comfortable with.

On a personal note I don't like JS for smart contracts, as it need to manage a lot of asynchronous stuff and it have some artifacs I don't like, but Python also has things I don't like so....
